modern/src/runtime/GuiObject.js

```diff
@@ -227,8 +228,8 @@ class GuiObject extends MakiObject {
   }
 
   settargeta(alpha) {
```

Comment:
Could we document the expected range in a comment here?

Reply:
will do